gems/pending/util/miq-ipmi.rb

```diff
@@ -209,11 +189,6 @@ def self.is_2_0_available?(ip_address)
     self.is_available_check(ip_address, "2.0")
   end
 
-  def self.is_1_5_available?(ip_address)
-    # One ping reply if machine supports IPMI V1.5
-    self.is_available_check(ip_address, "1.5")
-  end
-
```

Comment:
@jrafanie This call was not being used since calling `is_available_check` without a version will verify multiple interface versions.  The `is_2_0_available?` method is still used by `interface_mode` to determine what value to pass to impitool.